internal/pkg/template/templates/cicd/buildspec.yml

```diff
@@ -49,7 +49,7 @@ phases:
       # Check if the `svc package` commanded exited with a non-zero status. If so, echo error msg and exit.
       - >
         for env in $pl_envs; do
-          tag=$(sed 's/:/-/g' <<<"${CODEBUILD_BUILD_ID##*:}-${env}" | rev | cut -c 1-128 | rev)
+          tag=$(echo ${CODEBUILD_BUILD_ID##*:}-$env | sed 's/:/-/g' | rev | cut -c 1-128 | rev)
```

Comment:
Just to make sure was this tested in manually? do the tags in ECR look the same as before by default?

Reply:
yup yup!